scripts/exregional_run_analysis.sh

```diff
@@ -0,0 +1,594 @@
+  fi
+
+  n=$((n + 1))
+
+done
+
+
+#
+#-----------------------------------------------------------------------
+#
+# set default values for namelist
+#
+#-----------------------------------------------------------------------
+
+cloudanalysistype=0
+ifsatbufr=.false.
+ifsoilnudge=.false.
+beta1_inv=1.0
+ifhyb=.false.
+
+# Determine if hybrid option is available
+memname='atmf009'
+nummem=`more filelist03 | wc -l`
+nummem=$((nummem - 3 ))
+if [[ ${nummem} -eq 80 ]]; then
+  print_info_msg "$VERBOSE" "Do hybrid with ${memname}"
+  beta1_inv=0.15
+  ifhyb=.true.
+  print_info_msg "$VERBOSE" " Cycle ${YYYYMMDDHH}: GSI hybrid uses ${memname} with n_ens=${nummem}" 
+fi
+
+
+#
+#-----------------------------------------------------------------------
+#
+# link or copy background files
+#
+#-----------------------------------------------------------------------
+
+FV3SARPATH=${CYCLE_DIR}
+cp_vrfy ${fixdir}/fv3_akbk                               fv3_akbk
+cp_vrfy ${fixdir}/fv3_grid_spec                          fv3_grid_spec
+
+if [ ${BKTYPE} -eq 1 ]; then  # Use background from INPUT
+  cp_vrfy ${bkpath}/gfs_data.tile7.halo0.nc                gfs_data.tile7.halo0.nc_b
+  ${NCKS} -A -v  phis ${fixdir}/phis.nc                    gfs_data.tile7.halo0.nc_b
+  ${NCKS} -A -v radar_tten ${fixdir}/radar_tten_input.nc   gfs_data.tile7.halo0.nc_b
+
+  cp_vrfy ${bkpath}/sfc_data.tile7.halo0.nc        fv3_sfcdata
+  cp_vrfy gfs_data.tile7.halo0.nc_b                fv3_dynvars
+  ln_vrfy -s fv3_dynvars                           fv3_tracer
+
+  fv3sar_bg_type=1
+else
+  if [ ${DA_CYCLE_INTERV} -eq ${FCST_LEN_HRS} ]; then
+    restart_prefix=""
+  elif [ ${DA_CYCLE_INTERV} -lt ${FCST_LEN_HRS} ]; then
+    restart_prefix=${YYYYMMDD}.${HH}0000
+  else
+    print_err_msg_exit "\
+Restart hour should not larger than forecast hour:
+    Restart Hour = \"${DA_CYCLE_INTERV}\"
+    Forecast Hour = \"${FCST_LEN_HRS}\""    
+    exit 1
+  fi
+
+  cp_vrfy  ${bkpath}/${restart_prefix}.fv_core.res.tile1.nc             fv3_dynvars
+  cp_vrfy  ${bkpath}/${restart_prefix}.fv_tracer.res.tile1.nc           fv3_tracer
+  cp_vrfy  ${bkpath}/${restart_prefix}.sfc_data.nc                      fv3_sfcdata
+  ${NCKS} -A -v radar_tten ${fixdir}/radar_tten_restart.nc              fv3_tracer
+  fv3sar_bg_type=0
+fi
+
+# analysis time
+cp_vrfy ${fixdir}/fv3_coupler.res                        coupler.res.tmp
+cat coupler.res.tmp  | sed "s/yyyy/${YYYY}/" > coupler.res.newY
+cat coupler.res.newY | sed "s/mm/${MM}/"     > coupler.res.newM
+cat coupler.res.newM | sed "s/dd/${DD}/"     > coupler.res.newD
+cat coupler.res.newD | sed "s/hh/${HH}/"     > coupler.res.newH
+mv coupler.res.newH coupler.res
+rm coupler.res.newY coupler.res.newM coupler.res.newD
+
+# add radar tten array
+#cp_vrfy ${USHDIR}/addtten.py                        addtten.py 
+#/scratch1/BMC/wrfruc/Samuel.Trahan/soft/anaconda2-5.3.1/bin/python3.7 addtten.py fv3_tracer
+#python addtten.py fv3_tracer
+#
```

Comment:
Please remove the above commented code.

Reply:
I need to leave this part as a note for now. 

In the future, we need a better way to add then array.